### PR TITLE
Add e2e tests for kotlin, nodejs, php, python, ruby, and shell sample apps

### PR DESCRIPTION
**Pull request by chaodaiG**
Add more e2e tests for kotlin, nodejs, php, python, ruby, and shell sample apps

### CODE REVIEW

File: test/e2e/config.yaml [hunk 1]
```diff
@@ -18,3 +18,44 @@ languages:
...
+      - "Dockerfile"
+  - language: "nodejs"
+    expectedOutput: "Hello Node.js Sample v1!"
+    copies:
+      - "index.js"
+      - "package.json"
+      - "service.yaml"
+      - "Dockerfile"
+  - language: "php"
+    expectedOutput: "Hello PHP Sample v1!\\n"
```
Comment (adrcunha) [inline test/e2e/config.yaml]: I suggest your comparison ignores trailing blanks, LF or CR.

Comment (chaodaiG) [reply test/e2e/config.yaml]: The output is actually not a LF, it's a string `\n`, that's why I used double `\` here.

Comment (adrcunha) [reply test/e2e/config.yaml]: This doesn't seem right. Looks like an error in the sample. If so, please fix it.

Comment (chaodaiG) [reply test/e2e/config.yaml]: I do have a plan for consolidating all these outputs, will address this as well

**Pull request merged**